src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -307,7 +307,7 @@ Modelers:
     <value>Errors found during Swagger document validation.</value>
   </data>
   <data name="XMSPathBaseNotInPaths" xml:space="preserve">
-    <value>Paths in x-ms-paths must overload a normal path in the paths section.</value>
+    <value>Paths in x-ms-paths must overload a normal path in the paths section, i.e. a path in the x-ms-path must either be same as a path in the paths section or a path in the paths sections followed by additional parameters.</value>
```

Comment:
are we providing the user which path is violating the rule? I believe that's why we wanted to update the message, could you paste an example output of something violating this rule?